guides/v2.1/cloud/setup/first-time-setup.md

```diff
@@ -1,60 +1,54 @@
+{{site.data.var.ece}} environments are **Read Only**, including all Starter environments and all Pro Integration, Staging, and Production environments. In a local development environment, you can write and test code prior to pushing it to an Integration environment for further testing and deployment to Staging and Production. You must develop in a local workspace using a cloned Integration environment and push changes to the remote, read-only {{site.data.var.ece}} Git repository. You can choose one of two methods:
 
-## Project user account {#account}
+-  [Launch a Docker development environment](#docker)
+-  [Create a local development environment](#local)
 
-Depending on the [Project Owner]({{ page.baseurl }}/cloud/onboarding/onboarding-tasks.html), you may need to be invited by the owner to the project. This account gives you participation to the project and connects your local, credentials, and access to your company's {{site.data.var.ece}} account. The account includes all source code and deployment environments.
-
-After you receive an invitation, you can complete tasks to create your local, pull code, and deploy your store.
-
-## Helpful knowledge {#knowledge}
+## Gather credentials {#credentials}
 
-We highly recommend having experience with the following technologies and methodologies:
+Prior to setting up a workspace, gather the following credentials and accounts:
 
-* Magento [technology stack]({{ page.baseurl }}/architecture/tech-stack.html)
-* [Git](https://git-scm.com/){:target="_blank"} for repository and code management, [branching methodologies](https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows){:target="_blank"}, and working with Magento Git code
-* [GitHub](https://github.com/){:target="_blank"} for managing your code and accessing Magento repos
-* [Fastly](https://www.fastly.com/){:target="_blank"} for CDN and caching, used in Staging and Production
-* [Galera Cluster](http://galeracluster.com/){:target="_blank"} for databases, used in Staging and Production (with triple redundancy failover in Production)
+-  **Magento Access Key**  
+    Provides secure access through a 32-character authentication token. The token authenticates between your account, the Magento 2 code repo (repo.magento.com), and any other Git services like GitHub and 3rd party accounts needed for development. You can create multiple Magento access keys. For the workspace setup, you can start with one specific key for your code repository. To generate a key, contact the Project Owner to generate [Magento authentication keys]({{ page.baseurl }}/install-gde/prereq/connect-auth.html).
```

Comment:
I think we should be clear that repo.magento.com is . Composer repo. For example:

> The token authenticates between your account, the Magento 2 Composer repo (repo.magento.com)...